packages/debugger-extension/src/index.ts

```diff
@@ -180,13 +180,14 @@ const files: JupyterFrontEndPlugin<void> = {
 const notebooks: JupyterFrontEndPlugin<IDebugger.IHandler> = {
   id: '@jupyterlab/debugger-extension:notebooks',
   autoStart: true,
-  requires: [IDebugger, INotebookTracker, ITranslator],
+  requires: [IDebugger, INotebookTracker, ISessionContextDialogs, ITranslator],
```

Comment:
Similar to the command above: do we need `ISessionContextDialogs` to be required?